src/pybind/mgr/ansible/run-tox.sh

```diff
@@ -20,10 +27,11 @@ export CEPH_BUILD_DIR=$CEPH_BUILD_DIR
 source ${MGR_ANSIBLE_VIRTUALENV}/bin/activate
 
 if [ "$WITH_PYTHON2" = "ON" ]; then
-  ENV_LIST+="py27"
+  ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST+="py3"
+if [ "$WITH_PYTHON3" = "3" ]; then
+  ENV_LIST+="py3,"
 fi
+ENV_LIST=$(echo "$ENV_LIST" | sed -e 's/,$//')
```

Comment:
Here and after instead of invoking extra process (echo bash plus sed), you could use bash builtin functionality:
```
ENV_LIST=${ENV_LIST%,}
```

Reply:
@kshtsk TIL